Assets/HoloToolkit/Input/Scripts/Utilities/Managers/MixedRealityCameraManager.cs

```diff
@@ -62,29 +66,42 @@ public enum DisplayType
 
         private void Start()
         {
+            CurrentDisplayType = DisplayType.Opaque;
+
             if (!Application.isEditor)
             {
+                // Runtime HoloLens detection
 #if UNITY_WSA
 #if UNITY_2017_2_OR_NEWER
-            if (!HolographicSettings.IsDisplayOpaque)
+                if (!HolographicSettings.IsDisplayOpaque)
 #endif
                 {
                     CurrentDisplayType = DisplayType.Transparent;
-                    ApplySettingsForTransparentDisplay();
-                    if (OnDisplayDetected != null)
-                    {
-                        OnDisplayDetected(DisplayType.Transparent);
-                    }
-                    return;
                 }
 #endif
             }
 
-            CurrentDisplayType = DisplayType.Opaque;
-            ApplySettingsForOpaqueDisplay();
+#if UNITY_EDITOR
+            // Editor Hololens Remoting detection
+            var window = EditorWindow.GetWindow<HolographicEmulationWindow>(typeof(HolographicEmulationWindow));
```

Comment:
If users don't have the `HolographicEmulationWindow` open in the editor we never get a valid window.

Reply:
Hmm, just tested and the behavior is different. GetWindow creates a new window if there is no window open and it defaults to emulationmode.none.

Reply:
That's what I'm saying.

Reply:
Ok, so now we wait for Unity to expose something that may be more helpful?

Reply:
That would be nice, but I suppose this works in the meantime.

I'd just add a note in the comments or something.